utils/dependencies.py

```diff
@@ -252,6 +252,38 @@ def _CheckPythonModuleVersion(
     status_message = '{0:s} version: {1!s}'.format(module_name, module_version)
     return True, status_message
 
+  def _CheckLZMA(self):
+    """Checks the availability of lzma.
+
+    Returns:
+      tuple: consists:
```

Comment:
I haven't seen this way of describing tuples in the doc before. Is this in the styleguide?

Reply:
I meant "containing", thx for flagging. Regarding the style guide have a look at: https://github.com/log2timeline/l2tdocs/blob/master/process/Style-guide.md#multiple-return-types